### ISSUE TITLE
代码运行有问题

### ISSUE THREAD

**Issue by happy-nlp** [LopezGG/NN_NER_tensorFlow#8]
你好，感谢你的分享，让我有机会下载研究你的代码~
我在运行过程中遇到了问题，梯度值总是nan，只有第一次迭代可以正常显示loss，调整参数也没有解决问题，不知道你是否遇到，或能给我一些建议
另外我没有在项目中找到用于测试的数据，希望能分享下，感谢！

**Comment by happy-nlp** [comment]
我看到关闭的问题里面有人说是学习率的问题，我将学习率调整到1E-10，仍然有nan值，报错和他们的一样

**Comment by happy-nlp** [comment]
![image](https://user-images.githubusercontent.com/27487554/34308755-b85e8fae-e789-11e7-936e-633d5f434aae.png)

**Comment by happy-nlp** [comment]
I'm sorry I've entered Chinese before. I have tried to reduce the learning rate, increase the clipping, and still do not solve the Nan value problem. I even tuned the version of the TF to 1.1 that you were in line with, and it didn't work. I hope to reply in time, because I really need it now, I like the code you share very much, but I have been troubled by this operation problem for 10 days. I'm going crazy
